src/utilities/util_misc.c

```diff
@@ -8,20 +8,6 @@
 
 #include "util.h"
 
-#ifdef HAVE_LIBTCMALLOC
-/*
- * Include the TCMalloc header with the "-Wundef" diagnostic flag disabled. Compiling with strict
- * (where the 'Wundef' diagnostic flag is enabled), generates compilation errors where the
- * '__cplusplus' CPP macro is not defined. This being employed by the TCMalloc header to
- * differentiate C & C++ compilation environments. We don't want to define '__cplusplus' when
- * compiling C sources.
- */
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wundef"
-#include <gperftools/tcmalloc.h>
-#pragma GCC diagnostic pop
-#endif
-
```

Comment:
Really glad to see the utilities/ stuff getting dropped. That was always a bit of a hack.